class.jetpack.php

```diff
@@ -5130,6 +5130,7 @@ public function alert_auto_ssl_fail() {
 	 * @return string
 	 */
 	public static function xmlrpc_api_url() {
+		_deprecated_function( __METHOD__, 'jetpack-8.0' );
```

Comment:
Should we expand on this here, something like this?

```suggestion
		_deprecated_function( __METHOD__, 'jetpack-8.0', 'Automattic\\Jetpack\\Connection\\Manager::xmlrpc_api_url()' );
```